MicrosoftTeams.schema.json

```diff
@@ -110,6 +116,11 @@
                     "description": "A relative file path to a transparent PNG outline icon. The border color needs to be white. Size 20x20.",
                     "maxLength": 2048
                 },
+                "selectedOutline": {
```

Comment:
as per our discussion remove the selectedOutline and color the existing outline icon

Reply:
Coloring the outline icon looks ugly and doesn't match the hover state of our 1P app icons. We should introduce a new icon for this and use it when available. We can always fall back to the outline icon if this new one isn't provided.

Reply:
I also agree with @ydogandjiev  on this. I am also skeptical about colouring the icons ourselves. Because not all icons can be filled programatically to give the feel of the focused state. 

My main concern is. from an algorithm standpoint of view, if i want to create a focused state of the outline icon, the icon should look filled with color. So how do we find the pixels in the png which needs to be filled vs which should not be. Because pixels in the png other than the ones on the outline are simple transparent pixels with their rgba value 0,0,0,0. correct me me if i miss something here. 

@MattSFT  the function you were referring is using something called color matrix. If can transform the individual pixel values using certain conditions. In compose extension if you look at the youtube icon, if doesn't have highlighted state. I believe the gif and other icons are using different set of icons.

This is sample link to android documentation of color matrix. but the concept is same. https://developer.android.com/reference/android/graphics/ColorMatrix 
It operates on individual pixels to change the color of the pixel.

Reply:
I also tried using this https://developer.android.com/reference/android/graphics/PorterDuff.Mode

But the problem remains same, how to differentiate the transparent pixels outside the outline area from the ones inside the outline bounds in the png.